misc.sk

```diff
@@ -50,19 +50,34 @@ random items:
 	sugar = minecraft:sugar
 	ender pearl¦s = minecraft:ender_pearl
 	blaze rod¦s = minecraft:blaze_rod
-	nether wart = minecraft:nether_wart
+	nether wart¦s = minecraft:nether_wart
 	(ender eye¦s|eye¦s of ender) = minecraft:ender_eye
 	(bottle¦s o' enchanting|xp bottle¦s|experience bottle¦s) = minecraft:experience_bottle
 	fire charge¦s = minecraft:fire_charge
 	(book and quill¦s|writable book¦s) = minecraft:writable_book
-	# TODO maps?
+	[empty] map = minecraft:map
+	filled map = minecraft:filled_map
 	nether star¦s = minecraft:nether_star
-	# TODO fireworks
 	nether brick¦s = minecraft:nether_brick
 	nether quartz = minecraft:quartz
 	prismarine shard¦s = minecraft:prismarine_shard
 	prismarine crystal¦s = minecraft:prismarine_crystals
 	rabbit hide¦s = minecraft:rabbit_hide
+	
+{firework types}:
+	(0|zero) = {Fireworks:{Flight:0}}
+	(1|one) =  {Fireworks:{Flight:1}}
+	(2|two) = - {Fireworks:{Flight:2}}
+	(3|three) = - {Fireworks:{Flight:3}}
```

Comment:
Fairly sure variations need to be defined within a section, but maybe I'm wrong? @bensku

Reply:
Also, I'd recommend adding more natural wording for these just for the sake of it. Maybe for 0/1/2/3 it could be immediate/short/medium/long, respectively?

Reply:
They need to be in top level section.

Reply:
Yeah, then this needs to be moved into a section. It can just be named `fireworks` or something.

Reply:
I'll probably keep 0/1/2/3 since it's how it looks in-game.